OutOfSchool/OutOfSchool.ElasticsearchData/ESWorkshopProvider.cs

```diff
@@ -39,6 +40,43 @@ public override async Task<SearchResultES<WorkshopES>> Search(WorkshopFilterES f
         };
     }
 
+    public override async Task<PriceRangeES> GetPriceRangeAsync(WorkshopFilterES filter = null)
+    {
+        filter ??= new WorkshopFilterES();
+
+        var query = CreateQueryFromFilter(filter);
```

Comment:
here might be the same case as with database, price may be included in query filter so need a way (parameter) to not add it so you can reuse this query builder.